src/hcl/lexer.py

```diff
@@ -356,7 +363,14 @@ def t_error(self, t):
         else:
             _raise_error(t)
 
-    def __init__(self):
+    def __init__(self, export_comments=None):
+        if export_comments == 'LINE':
+            self.can_export_comments = ['COMMENT']
+        elif export_comments == 'MULTILINE':
+            self.can_export_comments = ['MULTICOMMENT']
+        elif export_comments == 'ALL':
+            self.can_export_comments = ['COMMENT', 'MULTICOMMENT']
```

Comment:
This should throw an exception if an invalid option is passed in, instead of failing silently.